Lombiq.DataTables.Samples/Views/Sample/DataTableTagHelper.cshtml

```diff
@@ -8,13 +7,6 @@
 
 @* Always make sure Views/_ViewImports.cshtml has "@addTagHelper *, Lombiq.DataTables" for the tag helper. *@
 
-@{
-    // Forces the CultureInfo to use the canonical date format. This prevents user or system level customizations. We
-    // need this for the UI tests to work in a portable way. This is a temporary workaround,  and the problem has been
-    // reported and a general solution requested here: https://github.com/OrchardCMS/OrchardCore/issues/11228
```

Comment:
Most possibly related to the `IgnoreSystemSettings` is this fail: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/4959155528

Reply:
Wait, no, that's in `dev`, I just got confused because I had a similar commit in the issue branch too. That seems to rather be something related to a GHA change...